bigtable/client/client_options.cc

```diff
@@ -17,7 +17,7 @@ namespace bigtable {
 inline namespace BIGTABLE_CLIENT_NS {
 // TODO() resolve issue #52
 ClientOptions::ClientOptions() {
-  char const* emulator = std::getenv("BIGTABLE_EMULATOR_HOST");
+  char const *emulator = std::getenv("BIGTABLE_EMULATOR_HOST");
```

Comment:
clang-format is permissive, and "deduces" the alignment for the `*`.  We are trying to consistently use `char const*`.  I will change the config in clang-format to enforce this, can you make it left aligned in this file please?

Reply:
Done.